updates/14.sql

```diff
@@ -0,0 +1,159 @@
+#provenance
+#
+#add a enum to associated migrated data with a source system (a source system can be for example dmp or batch import)
+#
+ALTER TABLE bemerkung ADD provenance VARCHAR (255);
```

Comment:
If we add multiple provenance-based fields, we should rename them:
- provenance => provenance_source, Default should be something like 'NeG' or 'neg'
- external_dmp_id => provenance_id (in the long term this could be something else than dmp, so we should not include the database name in the field), DEFAULT NULL (in case it is NeG we dont store any external ids here)

default values are important, else we cannot add new values anymore without changing lots of code